imageio/plugins/ffmpeg.py

```diff
@@ -250,8 +250,10 @@ def _get_cam_inputname(self, index):
                 # Ask ffmpeg for list of dshow device names
                 cmd = [self._exe, '-list_devices', 'true',
                        '-f', CAM_FORMAT, '-i', 'dummy']
+                # Set `shell=True` in sp.Popen to prevent popup of a command
+                # line window in frozen applications (Windows only).
                 proc = sp.Popen(cmd, stdin=sp.PIPE, stdout=sp.PIPE,
-                                stderr=sp.PIPE)
+                                stderr=sp.PIPE, shell=True)
```

Comment:
Did you omit `shell=is_win` here for a reason?

Also, please define `is_win` just once, near the top of the module.